src/ec/packages/ec-component-media-container/README.md

```diff
@@ -25,6 +25,7 @@ npm install --save @ecl-twig/ec-component-media-container
 - **"extra_attributes"** (optional) (array) (default: []) Extra attributes
   - "name" (string) Attribute name, eg. 'data-test'
   - "value" (string) Attribute value, eg: 'data-test-1'.
+- **"_compliance_"** (boolean) (default: false)
```

Comment:
Maybe we can have a small description like other component?
"compliance" (boolean) (default: false) Activates debug.

Reply:
sure ;)